addon/v1beta1/zz_generated.conversion.go

```diff
@@ -0,0 +1,929 @@
+	return nil
+}
+
+// Convert_v1alpha1_ClusterManagementAddOnStatus_To_v1beta1_ClusterManagementAddOnStatus is an autogenerated conversion function.
+func Convert_v1alpha1_ClusterManagementAddOnStatus_To_v1beta1_ClusterManagementAddOnStatus(in *v1alpha1.ClusterManagementAddOnStatus, out *ClusterManagementAddOnStatus, s conversion.Scope) error {
+	return autoConvert_v1alpha1_ClusterManagementAddOnStatus_To_v1beta1_ClusterManagementAddOnStatus(in, out, s)
+}
+
+func autoConvert_v1beta1_ConfigGroupResource_To_v1alpha1_ConfigGroupResource(in *ConfigGroupResource, out *v1alpha1.ConfigGroupResource, s conversion.Scope) error {
+	out.Group = in.Group
+	out.Resource = in.Resource
+	return nil
+}
+
+// Convert_v1beta1_ConfigGroupResource_To_v1alpha1_ConfigGroupResource is an autogenerated conversion function.
+func Convert_v1beta1_ConfigGroupResource_To_v1alpha1_ConfigGroupResource(in *ConfigGroupResource, out *v1alpha1.ConfigGroupResource, s conversion.Scope) error {
+	return autoConvert_v1beta1_ConfigGroupResource_To_v1alpha1_ConfigGroupResource(in, out, s)
+}
+
+func autoConvert_v1alpha1_ConfigGroupResource_To_v1beta1_ConfigGroupResource(in *v1alpha1.ConfigGroupResource, out *ConfigGroupResource, s conversion.Scope) error {
+	out.Group = in.Group
+	out.Resource = in.Resource
+	return nil
+}
+
+// Convert_v1alpha1_ConfigGroupResource_To_v1beta1_ConfigGroupResource is an autogenerated conversion function.
+func Convert_v1alpha1_ConfigGroupResource_To_v1beta1_ConfigGroupResource(in *v1alpha1.ConfigGroupResource, out *ConfigGroupResource, s conversion.Scope) error {
+	return autoConvert_v1alpha1_ConfigGroupResource_To_v1beta1_ConfigGroupResource(in, out, s)
+}
+
+func autoConvert_v1beta1_ConfigReference_To_v1alpha1_ConfigReference(in *ConfigReference, out *v1alpha1.ConfigReference, s conversion.Scope) error {
+	if err := Convert_v1beta1_ConfigGroupResource_To_v1alpha1_ConfigGroupResource(&in.ConfigGroupResource, &out.ConfigGroupResource, s); err != nil {
+		return err
+	}
+	out.LastObservedGeneration = in.LastObservedGeneration
+	out.DesiredConfig = (*v1alpha1.ConfigSpecHash)(unsafe.Pointer(in.DesiredConfig))
+	out.LastAppliedConfig = (*v1alpha1.ConfigSpecHash)(unsafe.Pointer(in.LastAppliedConfig))
+	return nil
+}
+
+// Convert_v1beta1_ConfigReference_To_v1alpha1_ConfigReference is an autogenerated conversion function.
+func Convert_v1beta1_ConfigReference_To_v1alpha1_ConfigReference(in *ConfigReference, out *v1alpha1.ConfigReference, s conversion.Scope) error {
+	return autoConvert_v1beta1_ConfigReference_To_v1alpha1_ConfigReference(in, out, s)
+}
+
+func autoConvert_v1alpha1_ConfigReference_To_v1beta1_ConfigReference(in *v1alpha1.ConfigReference, out *ConfigReference, s conversion.Scope) error {
+	if err := Convert_v1alpha1_ConfigGroupResource_To_v1beta1_ConfigGroupResource(&in.ConfigGroupResource, &out.ConfigGroupResource, s); err != nil {
+		return err
+	}
+	// WARNING: in.ConfigReferent requires manual conversion: does not exist in peer-type
```

Comment:
Should this also be handled by conversion.go? 
Conversion case is: 
https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/158-addon-v1beta1#managedclusteraddon-status